oak_remote_attestation_verification/src/verifier.rs

```diff
@@ -70,25 +75,46 @@ impl AttestationVerifier for InsecureAttestationVerifier {
     }
 }
 
-// Verifies the given BinaryAttestation object using the given Rekor and endorser public keys. Also
-// verifies that the given binary_digest is the same as the digest in the endorsement statement in
-// the input BinaryAttestation.
-pub fn verify_binary_attestation(
-    binary_digest: &[u8],
+/// Verifies the transparent release endorsement for a given measurement (which must be a SHA256
```

Comment:
Not needed to address in this PR, but for future design: What is the plan for endorsing the layer 0 measurement that is included in the attestation report? That is effectively a measurement of stage 0, but mixes in other things and, importantly, is a SHA2-384 measurement, not SHA2-256.